Marlin/Marlin_main.cpp

```diff
@@ -285,6 +288,14 @@ static int cmd_queue_index_w = 0;
 static int commands_in_queue = 0;
 static char command_queue[BUFSIZE][MAX_CMD_SIZE];
 
+#if ENABLED(INCH_MODE_SUPPORT)
+  float linear_unit_modifier = 1.0;
+  float volumetric_unit_modifier = 1.0;
```

Comment:
The term "modifier" implies addition or subtraction. I think the term "factor" would be better than "modifier" here, and more consistent with usage elsewhere in Marlin.